### ISSUE TITLE
Alt Attribute

### ISSUE THREAD

**Issue by jim-rosales** [SEANPACKARD/prj-rev-bwfs-dasmoto#3]
Don't forget to include the [alt attribute](https://en.wikipedia.org/wiki/Alt_attribute)! The alt attribute increases the [accessibility](https://www.codecademy.com/articles/accessibility) of your page, optimizes it for search engine results, and is used as a backup, in the chance that the image cannot be rendered. 

https://github.com/SEANPACKARD/prj-rev-bwfs-dasmoto/blob/bf60bf04c1ff8f8aa63a5ac54d7f5e9acc2a63e2/Crafts/index.html#L23
